zombienet/0001-basic-warp-sync/0001-test-warp-sync.zndsl

```diff
@@ -0,0 +1,30 @@
+Description: Warp sync
+Network: ./0001-test-warp-sync.toml
+Creds: config
+
+alice: is up
+bob: is up
+charlie: is up
+dave: is up
+
+alice: reports node_roles is 4
+bob: reports node_roles is 4
+charlie: reports node_roles is 1
+dave: reports node_roles is 1
+
+alice: reports peers count is at least 3
+bob: reports peers count is at least 3
+charlie: reports peers count is at least 3
+dave: reports peers count is at least 3
+
+dave: log line matches "Warp sync is complete" within 5 seconds
+dave: log line matches "Block history download is complete" within 5 seconds
+
+# db snapshot has 12133 blocks
+dave: reports block height is at least 1 within 5 seconds
+dave: reports block height is at least 12132 within 5 seconds
+dave: reports block height is at least 12133 within 5 seconds
+
+alice: reports block height is at least 12133 within 5 seconds
+bob: reports block height is at least 12133 within 5 seconds
+charlie: reports block height is at least 12133 within 5 seconds
```

Comment:
These timeouts seem to be too tight. CI systems tend to be overloaded resulting in requiring more time :P

Reply:
Increased to `30s` in [0649849](https://github.com/paritytech/substrate/pull/12675/commits/0649849bd2088cfe774b1444b24a0ac4e388e9d2)